### ISSUE TITLE
Camera position is far off, as well as controller

### ISSUE THREAD

**Issue by KEWLIOSUCKA** [cboulay/PSMoveService#119]
Alright I hope this is legit and I'm not just being my dumb self xD I've calibrated everything correctly, made sure the camera can see the bulb perfectly, HMD tracking is great, all that good stuff. Problem starts when trying to computer tracker poses. I hold the controller firmly to the side (I'll get to that in a second) of my HMD (DK2 if it makes much of a difference), but when it's finished the controller position is way off, and the camera is as well. The camera position doesn't seem to change unless I exit and reopen the app, is it supposed to change after calibration? I'm guessing no but better to make sure. Granted I'm simply holding the controller to the DK2, but I've held it as firmly as possible and even flailed it wildly about and still get not big differences in results. If the smallest of movements could cause such big offsets, I'll admit fault and superglue the friggen thing to the HMD xD There was ONE TIME it was pretty much accurate, but it still wasn't *that* accurate so I tried calibrating again. Oh the regrets... Currently I have the Oculus tracker and my PS Eye sitting next to each other on my dresser, overlooking my play area. Could that be causing the problems as well? Thanks ^-^

**Comment by cboulay** [comment]
Can you confirm the colour mask is working well in the color calibration tool?

Don't flail about. Move purposefully and slowly, always keeping the bulb in complete view of the camera. 60 fps is pretty good but it's still susceptible to motion blur.

Can you try using the calibration mat method? I've never tried it but I presume it also works with the DK2?

**Comment by KEWLIOSUCKA** [comment]
Yup, it stands out against the background and the bulb is defined in all three.

I've tried it all. Slowly in every direction I can, and the flailing like I said. Both produce roughly the same results.

Just printed one out. Got quite different results, but still way off position. I put a tall object on top of my dresser to elevate the cameras, and to make sure they completely got me and the floor. Tried the calibration mat, then the PS eye was behind me in the demo area. Re-ran the HMD attaching option and I got the same results as in my first post. The controller itself is tracking wonderfully most of the time, but it's just not in the right position.

**Comment by cheddle** [comment]
Im getting the same problem with the PSeye being 1 meter in front of the DK2 camera (they are taped together)

**Comment by cheddle** [comment]
Came back to say that recalibrate resolved. I imagine two cameras will sort this 100%

**Comment by KEWLIOSUCKA** [comment]
Huh, well I guess it isn't as whacked out as I thought. Two cameras does seem like it would help a lot. They're only $5 on Amazon so I'll see if I can get any soon and test with multiple.
